.github/workflows/ci.yml

```diff
@@ -79,9 +79,9 @@ jobs:
         with:
           path: |
             packages/*/lib/*
-          key: ${{ runner.os }}-${{ hashFiles('yarn.lock') }}
+          key: ${{ runner.os }}-${{ hashFiles('yarn.lock') }}-1
           restore-keys: |
-            ${{ runner.os }}-yarn-
+            ${{ runner.os }}-yarn-1
```

Comment:
Is this intentional, or was it accidentally left in?

Reply:
nope intentional. just posted an async explaining this: https://github.com/dendronhq/dendron-docs/blob/main/vault/dev.process.code.gotchas.changing-file-to-folder.md#L16:L16